hardware_interface/test/test_components/test_sensor.cpp

```diff
@@ -15,19 +15,25 @@
 #include <memory>
 #include <vector>
 
+#include "hardware_interface/components/base_interface.hpp"
 #include "hardware_interface/components/sensor_interface.hpp"
 
 using hardware_interface::status;
 using hardware_interface::return_type;
 using hardware_interface::StateInterface;
 
-class TestSensor : public hardware_interface::components::SensorInterface
+class TestSensor : public
+  hardware_interface::components::BaseInterface<hardware_interface::components::SensorInterface>
```

Comment:
I'd strongly suggest dropping some `using`-s to reduce the length of lines like this

Reply:
What do you mean exactly?

Reply:
Ideally you'd want to see a line like this:

`class TestSensor : public BaseInterface<SensorInterface>`